### ISSUE TITLE
Required inputs should have html 'required' attribute

### ISSUE THREAD

**Issue by rogercoder** [ngx-formly/ngx-formly#1582]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**

If an input is required, the label shows the asterisk but the input element does not include the 'required' html attribute.

**Expected behavior**

If an input is required then the input element should have the html 'required' attribute.

**Minimal reproduction of the problem with instructions**

The base ngx-formly bootstrap stackblitz shows this:
https://stackblitz.com/edit/ngx-formly-ui-bootstrap

Right click on the required email field and the input tag does not have the 'required' attribute.

**What is the motivation / use case for changing the behavior?**

For ARIA/WCAG compliance, required inputs should have the 'required' attribute.
If this attribute is used on any form field it is identified as required by assistive technologies. 
See, for just one example, a discussion at http://john.foliot.ca/required-inputs/

**Please tell us about your environment:**

Windows 10, WebStorm

* **Angular version:** 2.0.X

Angular 7

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

Chrome 73
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
TypeScript 3.2.4
* **Node (for AoT issues):** `node --version` =
10.15.1

**Comment by aitboudad** [comment]
I'll check it later, in the meantime, you can use `attributes` property as shown in the following example https://stackblitz.com/edit/ngx-formly-ui-bootstrap-rsg5x1

**Comment by rogercoder** [comment]
Thanks! I'll do that ... however according to the HTML 5 spec for boolean attributes such as 'required': 

> If the attribute is present, its value must either be the empty string or a value that is a case-insensitive match for the attribute’s canonical name, with no leading or trailing whitespace. 

(from _3.2.2 Boolean attributes_ at [W3C HTML 5 docs](https://www.w3.org/TR/2008/WD-html5-20080610/semantics.html)

So it should be:
`      attributes: {
        required: 'required'
      }`
rather than setting the value to be 'true'.

**Comment by aitboudad** [comment]
fixed in `5.1.7`
